### PR TITLE
Updated JavaDoc comments to eliminate warnings and errors that were p…

### PR DESCRIPTION
**Pull request by brian-rickman**
…rinted during the build process.

### CODE REVIEW

**Comment by kokosing** [comment]
Looks good :)

File: tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/TableRequirements.java [hunk 1]
```diff
@@ -37,6 +45,9 @@ public static MutableTableRequirement mutableTable(TableDefinition tableDefiniti
 
     /**
      * Requirement for immutable table.
+     *
+     * @param tableDefinition Encapsulated DDL for table
+     *                        @return ImmutableTableRequirement
```
Comment (losipiuk) [inline tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/TableRequirements.java]: indentation

File: tempto-core/src/main/java/com/teradata/tempto/internal/configuration/MapConfiguration.java [hunk 2]
```diff
@@ -29,21 +29,26 @@
...
  * For example, for a configuration storing these hierarchical entries:
  * a.b.c = 3
  * a.x = 10
  * b.c = 15
- * <p/>
+ * </P>
+ * <P>
  * the following map structure would be created
- * <p/>
+ * </P>
```
Comment (losipiuk) [inline tempto-core/src/main/java/com/teradata/tempto/internal/configuration/MapConfiguration.java]: lowercase

File: tempto-logging-log4j/src/main/java/com/teradata/logging/TestFrameworkLoggingAppender.java [hunk 3]
```diff
@@ -176,6 +174,8 @@ public boolean requiresLayout()
 
     /**
      * Returns logs directory for configured TestFrameworkLoggingAppender.
```
Comment (losipiuk) [inline tempto-logging-log4j/src/main/java/com/teradata/logging/TestFrameworkLoggingAppender.java]: maybe remove summary as \@return states the same

**Comment by losipiuk** [comment]
looks good - thanks

**Comment by kokosing** [comment]
So now once warnings are fixed, maybe we could change them into errors, so
when somebody makes a mistake it will cause build to fail.

czw., 25.06.2015 o 12:51 użytkownik Łukasz Osipiuk <notifications@github.com>
napisał:

**Comment by brian-rickman** [comment]
I have addressed the comments and updated build.gradle so that the build will fail for JavaDoc errors.

**Comment by losipiuk** [comment]
cool :) than ks

**Pull request merged**